refractr/Dockerfile

```diff
@@ -4,7 +4,8 @@ ARG NGINX_VERSION=1.17.6
 FROM nginx:alpine AS builder
 LABEL stage=intermediate
 
-ENV NGINX_VERSION 1.17.6
```

Comment:
I thought we wanted ENVs not ARGs after the FROM blah AS foo line?

Reply:
we chatted ARG before and after the FROM